src/components/FeatureViews/Accounts/OutOfOffice.js

```diff
@@ -74,167 +75,172 @@ const OutOfOffice = ({instructorID, button}) => {
-                        item
-                        md={3}>
-                        <div className="select">
-                            * Select OOO End Date
-                        </div>
-                        <DatePicker
-                            animateYearScrolling
-                            format="MM/dd/yyyy"
-                            label="End Date"
-                            margin="normal"
-                            onChange={setEnd}
-                            openTo="day"
-                            value={end}
-                            views={["year", "month", "date"]} />
-                    </Grid>
-                    <Grid
-                        item
-                        md={6} />
-                    <Grid
-                        item
-                        md={3}>
-                        <div className="select">
-                            * Select OOO Start Time
-                        </div>
-                        <TimePicker
-                            autoOk
-                            disabled={allDay}
-                            label="Start Time"
-                            value={start}
-                            onChange={setStart} />
-                    </Grid>
+                    </div>
+                    <TextField
+                        label="Description"
+                        onChange={updateDescription}
+                        value={description} />
                     <Grid
+                        container
                         item
-                        md={3}>
-                        <div className="select">
-                            * Select OOO End Time
-                        </div>
-                        <TimePicker
-                            autoOk
-                            disabled={allDay}
-                            label="End Time"
-                            value={end}
-                            onChange={setEnd} />
-                    </Grid>
-                    <Grid
-                        item
-                        md={2}>
-                        <Grid container>
-                            <Grid>
-                                <Checkbox
-                                    checked={allDay}
-                                    className="checkbox"
-                                    inputProps={{
-                                        "aria-label": "primary checkbox",
-                                    }}
-                                    onChange={toggleAllDay}
-                                    value="primary" />
-                            </Grid>
-                            <Grid>
-                                <div className="checkboxText">
+                        md={12}>
+                        <Grid
+                            item
+                            md={3}>
+                            <div className="select">
+                                * Select OOO Start Date
```

Comment:
I thought this was done a while ago, but remove the asterix, and change the field to be required